packages/core/test/Hug.js

```diff
@@ -44,7 +45,7 @@ describe('Hug', () => {
 
   test('renders text when array of nodes is provided', () => {
     const json = rendererCreateWithTheme(
-      <Hug icon="thumbsUp" text={[text, text]}>
+      <Hug icon={<ThumbsUp />} text={[text, text]}>
```

Comment:
Is there a method for adding an icon to Hug that will work for both v2 and v3?

Reply:
I don't think so, this is our major breaking change that the whole v3 release was designed around.

Reply:
There's no reason you couldn't pass `<Icon name="ThumbsUp">`, it just won't be tree-shakable.

Reply:
If you're passing an icon node that means you're on v3 of design-system, so there would be no need to pass Icon at that point.

Reply:
If we decide it's worth the effort, perhaps those changes can be backported - so the components in question will prefer the `icon` prop if provided but fall back to `iconName`.